Example/Example/AppDelegate.swift

```diff
@@ -2,169 +2,177 @@
 class AppDelegate: UIResponder, UIApplicationDelegate {
+    var locationManager = CLLocationManager()
     var window: UIWindow?
-    
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Pass your account token from the Rover Settings app to the Rover SDK.
-        Rover.accountToken = "<YOUR_SDK_TOKEN>"
 
-        // This method demonstrates how to observe the Rover events mentioned above in your own app.
-        observeRoverNotifications()
-        return true
-    }
-    
-    // This app delegate method is called when any app (your own included) calls the
-    // `open(_:options:completionHandler:)` method on `UIApplication` with a URL that matches one of the schemes setup
-    // in your `Info.plist` file. These custom URL schemes are commonly referred to as "deep links". This Example app
-    // uses a custom URL scheme `example` which is configured in Example/Example/Info.plist.
-    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
-        // You will need to setup a specific URL structure to be used for presenting Rover experiences in your app. The
-        // simplest approach is to use a specific URL path/host and include the experience ID and (optional) campaign
-        // ID as query parameters. The below example demonstrates how to route URLs in the format
-        // `example://experience?id=<EXPERIENCE_ID>&campaignID=<CAMPAIGN_ID>` to a Rover experience.
-        if url.host == "experience" {
-            guard let queryItems = URLComponents(url: url, resolvingAgainstBaseURL: false)?.queryItems else {
-                return false
-            }
-            
-            guard let id = queryItems.first(where: { $0.name == "id" })?.value else {
-                return false
-            }
-            
-            let campaignID = queryItems.first(where: { $0.name == "campaignID" })?.value
-            
-            // you can optionally pass in a screen ID to navigate to immediately rather than starting at the home screen.
-            let screenID = queryItems.first(where: { $0.name == "screenID" })?.value
-            
-            // Instantiate a `RoverViewController` and call the `loadExperience(id:campaignID)` method to load the
-            // experience whose id matches the value passed in the query parameter.
-            let roverViewController = RoverViewController()
-            roverViewController.loadExperience(id: id, campaignID: campaignID, initialScreenID: screenID)
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Initialize the Rover SDK with all modules.
+        RoverFoundation.initialize(assemblers: [
+            FoundationAssembler(),
+            DataAssembler(accountToken: "be40ec7dabf7c9a06a451e58a4c39d9823c24974"),
```

Comment:
you committed your token by mistake here

Reply:
Ooops.  Fixed.